artifactory/commands/transferconfig/transferconfig_test.go

```diff
@@ -169,6 +177,34 @@ func TestVerifyConfigImportPluginForbidden(t *testing.T) {
 	assert.ErrorContains(t, err, "Response from Artifactory: 403 Forbidden.")
 }
 
+func TestSetExportPath(t *testing.T) {
+	transferConfigBase := NewTransferConfigCommand(nil, nil)
+
+	// Create export path and check results
+	exportPath, unsetTempDir, err := transferConfigBase.createExportPath()
+	unsetTempDir()
+	assert.NoError(t, err)
+	assert.NotEmpty(t, exportPath)
+	assert.DirExists(t, exportPath)
+
+	// Create working dir
+	tmpDir, createTempDirCallback := tests.CreateTempDirWithCallbackAndAssert(t)
+	defer createTempDirCallback()
+	tmpDir = filepath.Join(tmpDir, "test-export-dir")
```

Comment:
Let's define "test-export-dir" as a const inside this function. beucase it's used twice inside the function.